scripts/compose.py

```diff
@@ -1550,6 +1550,7 @@ class LocalSetup(object):
         '6.5': '6.5.4',
         '6.6': '6.6.1',
         '6.7': '6.7.0',
+        'master': '6.6.1',
```

Comment:
I think is not needed, the build is using the parameter ELASTIC_STACK_VERSION="6.6 --release"

Reply:
Ok, I'll revert that